core/src/main/java/cz/o2/proxima/storage/URIUtil.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2017 O2 Czech Republic, a.s.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.o2.proxima.storage;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URI;
+import java.net.URLDecoder;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Utilities related to URIs.
+ */
+public class URIUtil {
```

Comment:
Is there any reason, you are you still using URIs for configuration? I think this could be done in much convenient way

Reply:
Would you mind filling issue describing the solution? We can have a conversation about that there. Thanks! :+1: